samples/nrf9160/modem_shell/src/utils/mosh_print.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
```

Comment:
Is `assert.h` needed?

Reply:
Wasn't needed but I actually made one assert why we need this include now.